<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plotter_Cfg</class>
 <widget class="QWidget" name="Plotter_Cfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="gbxTcpIp">
     <property name="title">
      <string>Tcp/Ip server</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnStartListen">
          <property name="text">
           <string>Start Listen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnStopListen">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stop Listen</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;amp;Port:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="buddy">
             <cstring>edtPort</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPort">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>4545</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="edtLog">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="btnCleanLog">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Clean log</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbxConfigure">
     <property name="title">
      <string>Limits</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Size of window to store points per plot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbMaximumPointsPerPlot">
          <property name="specialValueText">
           <string/>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
          <property name="value">
           <number>1000000</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbxStat">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Stat</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="lblClients">
        <property name="text">
         <string>Current Established connections:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="edtClientsCount">
        <property name="text">
         <string>0</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblProcessCommands">
        <property name="text">
         <string>Number of processed commands</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="edtProcessCommands">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblProcessedKBytes">
        <property name="text">
         <string>Number of processed Kbytes</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="edtProcessedKBytes">
        <property name="text">
         <string>0</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btnCleanStat">
        <property name="text">
         <string>Clean stat</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnStartListen</tabstop>
  <tabstop>btnStopListen</tabstop>
  <tabstop>edtPort</tabstop>
  <tabstop>edtLog</tabstop>
  <tabstop>btnCleanLog</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnStartListen</sender>
   <signal>clicked()</signal>
   <receiver>Plotter_Cfg</receiver>
   <slot>slotStartListen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStopListen</sender>
   <signal>clicked()</signal>
   <receiver>Plotter_Cfg</receiver>
   <slot>slotStopListen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCleanLog</sender>
   <signal>clicked()</signal>
   <receiver>Plotter_Cfg</receiver>
   <slot>slotCleanLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotStartListen()</slot>
  <slot>slotStopListen()</slot>
  <slot>slotCleanLog()</slot>
 </slots>
</ui>
